src/api/fetchConversationList.ts

```diff
@@ -0,0 +1,42 @@
+import axios, { AxiosError } from "axios";
+import { apiUrl } from "@/utils/contants";
+import { headerObject, Conversation } from "@/types";
+
+const FETCH_CONVERSATION_LIST_URL = `${apiUrl}fetchConversationList/`;
+
+const BASE_HEADERS: Record<string, string> = {
+      "Content-Type": "application/json",
+      "Access-Control-Allow-Origin": "*",
+      "Access-Control-Allow-Methods": "POST,OPTIONS",
```

Comment:
CORS headers incorrectly set in request instead of server response, causing potential security/access issues.